internal/mode/static/state/graph/policies.go

```diff
@@ -203,22 +251,8 @@ func processPolicies(
 			continue
 		}
 
-		for _, targetedRoute := range targetedRoutes {
```

Comment:
Had to extract a function here to satisfy the cyclo complexity linter. @sjberman can you double check this to make sure I didn't mess anything up?